### ISSUE TITLE
[1.4.5] Right click functions for off hand items blocked.

### ISSUE THREAD

**Issue by Hexblood** [Direwolf20-MC/JustDireThings#242]
### Just Dire Things version

1.4.5

### Minecraft Version

1.21.1

### (Neo)Forge Version

21.1.71

### Do you have optifine or Rubidium installed?

I have other rendering based mods installed

### Describe the issue

Right-clicking with JDT tools in main hand does not right-click use item in off hand.

### Steps to reproduce

1. Install or update JDT to version 1.4.5 in your modpack.
2. Attempt to right-click use/place item from offhand while JDT tool is in main-hand. (Found with Blazegold tier in survival world. Tested with all 4 tiers in test world.)

### Expected behaviour

Items to be used normally from off hand.

### Additional information

This issue is new as of version 1.4.5.

**Comment by Direwolf20-MC** [comment]
Thanks for the report ;). I changed some mechanics to this in 1.4.5 and must have added a bug. 

You mean like placing a torch from your off hand right ?

**Comment by Hexblood** [comment]
Yes, tested with both placing a torch and casting spells from Ars Book in the offhand.
